src/lib/rest/OrionError.cpp

```diff
@@ -161,8 +161,8 @@ std::string OrionError::render(ConnectionInfo* ciP, const std::string& _indent)
 
   if (format == NOFORMAT)
   {
-    // Default format is XML for "v1"
-    format = XML;
+    // Default format is JSON
+    format = JSON;
   }
 
```

Comment:
Yes, it seems we are still using format in the NGSIv2 for text rendering. From getEntityAttributeValue() function:

```
  bool         text       = (ciP->uriParamOptions["options"] == true || ciP->outFormat == TEXT);
```

(Side-note: the `ciP->uriParamOptions["options"] == true` part should be removed. It corresponds to an older version of the NGSIv2, no longer supported)